modules/skybrowser/skybrowsermodule_lua.inl

```diff
@@ -765,6 +765,17 @@ namespace {
         );
 }
 
+/**
+ * Stop animations
+ * \param identifier Identifier of the sky browser
```

Comment:
The way we are parsing these comments for the Lua file doesn't follow the Doxygen way yet. So you'll have to write the description out at the moment:
```
/**
 * Stop animations.  The parameter to this function is the identifier of the sky browser
 */
```
or something like that